maestro-client/src/main/java/maestro/Maestro.kt

```diff
@@ -534,11 +505,66 @@ class Maestro(private val driver: Driver) : AutoCloseable {
         return driver.isUnicodeInputSupported()
     }
 
+    private fun waitUntilScreenIsStatic(timeoutMs: Long): Boolean {
+        if (isIOS) {
+            return MaestroTimer.retryUntilTrue(timeoutMs) {
+                val screenChanged = driver.isScreenChanged()
+
+                LOGGER.info("screen changed = $screenChanged")
+                return@retryUntilTrue !screenChanged
+            }
+        } else {
+            return MaestroTimer.retryUntilTrue(timeoutMs) {
+                val startScreenshot: BufferedImage? = tryTakingScreenshot()
+                val endScreenshot: BufferedImage? = tryTakingScreenshot()
+
+                if (startScreenshot != null &&
+                    endScreenshot != null &&
+                    startScreenshot.width == endScreenshot.width &&
```

Comment:
Why are we comparing width and height? Will not that always be same?

Reply:
the logic was there already, I think it is because ImageComparison library expects it to be the same, we just catch this